subscriber/podaac_access.py

```diff
@@ -113,23 +134,45 @@ def get_token(url: str, client_id: str, endpoint: str) -> str:
 ###############################################################################
 # DELETE TOKEN FROM CMR
 ###############################################################################
-def delete_token(url: str, token: str) -> None:
+def delete_token(url: str, token: str) -> bool:
     try:
-        headers: Dict = {'Content-Type': 'application/xml', 'Accept': 'application/json'}  # noqa E501
-        url = '{}/{}'.format(url, token)
-        resp = requests.request('DELETE', url, headers=headers)
-        if resp.status_code == 204:
-            logging.info("CMR token successfully deleted")
+        username, _, password = netrc.netrc().authenticators(edl)
+        headers: Dict = {'Accept': 'application/json'}
+        resp = requests.post(url+"/revoke_token",params={"token":token}, headers=headers, auth=HTTPBasicAuth(username, password))
+
+        if resp.status_code == 200:
+            logging.info("EDL token successfully deleted")
+            return True
         else:
-            logging.info("CMR token deleting failed.")
+            logging.info("EDL token deleting failed.")
+
     except:  # noqa E722
         logging.warning("Error deleting the token")
 
+    return False
+
+def list_tokens(url: str):
+    try:
+        tokens = []
+        username, _, password = netrc.netrc().authenticators(edl)
+        headers: Dict = {'Accept': 'application/json'}  # noqa E501
+        resp = requests.get(url+"/tokens", headers=headers, auth=HTTPBasicAuth(username, password))
+        response_content = json.loads(resp.content)
+
+        for x in response_content:
+            tokens.append(x['access_token'])
+
+    except:  # noqa E722
```

Comment:
If you want to be more specific here, check https://stackoverflow.com/a/16511493